external_api/constants.py

```diff
@@ -0,0 +1,5 @@
+DICTIONARY_NAME = 'dictionary_name'
```

Comment:
There is no concept of specific dictionary_name in entire Repo. All detectors get initialized by entity_name itself. For consistency, same should be the case with external apis also.